backend/census_historical_migration/workbooklib/federal_awards.py

```diff
@@ -155,7 +163,24 @@ def xform_replace_invalid_extension(audit):
     if not is_valid_extension(extension):
         extension = settings.GSA_MIGRATION
 
-    return f"{prefix}.{extension}"
+    cfda_key = f"{prefix}.{extension}"
+    census_data = [
+        CensusRecord("CFDA_PREFIX", audit.CFDA_PREFIX).to_dict(),
+        CensusRecord("CFDA_PREFIX", audit.CFDA_EXT).to_dict(),
+    ]
+    gsa_fac_data = GsaFacRecord("cfda_key", cfda_key).to_dict()
```

Comment:
The target field is `federal_award_extension` not `cfda_key`. So this would be `GsaFacRecord("federal_award_extension", cfda_key).to_dict()`

Reply:
`CensusRecord("CFDA_PREFIX", audit.CFDA_PREFIX).to_dict()` is not necessary since we are not making change to the prefix.
Here, it should be ` CensusRecord("CFDA_EXT", audit.CFDA_EXT).to_dict()`  instead of ` CensusRecord("CFDA_PREFIX", audit.CFDA_EXT).to_dict()`